### ISSUE TITLE
mcu 'mcu': Unable to connect after power cycle

### ISSUE THREAD

**Issue by that-should-work** [Klipper3d/klipper#4910]
[klippy.log](https://github.com/Klipper3d/klipper/files/7514034/klippy.log)
Line 244
Klipper/Fluidd has been running fine until the printer got power cycled

mcu 'mcu': Unable to connect
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, and restart the host software.
Error configuring printer

Tried using 
ls /dev/serial/by-id/*
but no such file or directory

pi@fluiddpi:~/klipper $ lsusb
Bus 001 Device 005: ID 046d:0843 Logitech, Inc. Webcam C930e
Bus 001 Device 004: ID 0424:7800 Standard Microsystems Corp.
Bus 001 Device 003: ID 0424:2514 Standard Microsystems Corp. USB 2.0 Hub
Bus 001 Device 002: ID 0424:2514 Standard Microsystems Corp. USB 2.0 Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

This is not the first time it's been power cycled, so I'm not sure what direction to take.

**Comment by MrSkrifle** [comment]
Try running:
cd ~/klipper/

Before:
ls /dev/serial/by-id/*

It should show then, but probably it hasn't changed. 


Try disconnecting power to pi and printer, and then try to power on. Repeat a few times. I get the same issue too often if I restart pi or klipper firmware through octoprint.

**Comment by that-should-work** [comment]
So I got it figured out just last night.
Changing to the klipper folder didn't work btw

I turned off power to the printer then connected the pi which powered the board. I was only able to connect then and I reflashed the board to make sure it was on the same page with all the klipper updates that the pi was on. Did a firmware restart and we're good to go.

Mostly...
pin_map is no longer used, but klipper has the Marlin pin conversion table for most boards. I used that to clean up the config file.
